src/common/services/api/donations.service.js

```diff
@@ -74,12 +74,12 @@ function DonationsService( cortexApiService, profileService, commonService ) {
         data:           data
       } )
       .map( ( response ) => {
-        angular.forEach( response['receipt-summaries'], ( item ) => {
-          let link = find( response.links, ( r ) => {
-            return r.uri.indexOf( item['transaction-number'] ) != -1;
-          } );
-          item['pdf-link'] = link;
-        } );
+        let links = filter(response.links, (link) => {
+          return link.rel == 'element';
+        }) || [];
```

Comment:
I think filter always returns an array. I tried `_.filter(undefined, () => false)` in the console on the lodash docs page and it returns `[]`.